perception/image_projection_based_fusion/config/segmentation_pointcloud_fusion.param.yaml

```
@@ -1,39 +1,24 @@
 /**:
   ros__parameters:
     # if the semantic label is applied for pointcloud filtering
+
     filter_semantic_label_target:
-      [
-      true, # road
-      true, # sidewalk
-      true, # building
-      true, # wall
-      true, # fence
-      true, # pole
-      true, # traffic_light
-      true, # traffic_sign
-      true, # vegetation
-      true, # terrain
-      true, # sky
-      false, # person
-      false, # ride
-      false, # car
-      false, # truck
-      false, # bus
-      false, # train
-      false, # motorcycle
-      false, # bicycle
-      false, # others
-      ]
-    # the maximum distance of pointcloud to be applied filter,
-    # this is selected based on semantic segmentation model accuracy,
-    # calibration accuracy and unknown reaction distance
-    filter_distance_threshold: 60.0
+      UNKNOWN: false
+      BUILDING: true
+      WALL: true
+      OBSTACLE: false
+      TRAFFIC_LIGHT: false
+      TRAFFIC_SIGN: false
+      PERSON: false
+      VEHICLE: false
+      BIKE: false
+      ROAD: true
+      SIDEWALK: false
+      ROAD_PAINT: false
+      CURBSTONE: false
+      CROSSWALK: false
+      VEGETATION: true
+      SKY: false
 
-    # debug
-    debug_mode: false
-    filter_scope_min_x: -100.0
-    filter_scope_max_x: 100.0
-    filter_scope_min_y: -100.0
-    filter_scope_max_y: 100.0
-    filter_scope_min_z: -100.0
-    filter_scope_max_z: 100.0
+    # the maximum distance of pointcloud to be applied filter
```

I can not get what `distance` means until reading the code.
It means maximum z depth, so how about change its explanation? 

```suggestion
    # the maximum depth of pointcloud to be applied filter
```